src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitException.java

```diff
@@ -17,7 +17,10 @@
 public class ShortCircuitException extends RuntimeException {
```

Comment:
Must implement RestResponseEntityExceptionHandler to return correct http code for each type of ShortCircuitException, for example 

@ControllerAdvice
public class RestResponseEntityExceptionHandler {
@ExceptionHandler(SensibilityAnalysisException.class)
protected ResponseEntity<Object> handleSensibilityAnalysisException(SensibilityAnalysisException exception) {
return switch (exception.getType()) {
case RESULT_NOT_FOUND -> ResponseEntity.status(HttpStatus.NOT_FOUND).body(exception.getMessage());
case FILE_EXPORT_ERROR -> ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(exception.getMessage());
case INVALID_EXPORT_PARAMS -> ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exception.getMessage());
default -> ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
};
}
}

Reply:
done